appveyor.yml

```diff
@@ -37,22 +37,27 @@ install:
 before_build:
 - cmd: SET PATH=%CONDA_PREFIX%;%CONDA_PREFIX%\Scripts;%PATH%
 - cmd: SET _prefix=%CONDA_PREFIX:~0,14%
+- cmd: SET _arch=%CONDA_PREFIX:~15,18%
 - >
-  IF "%_prefix%"=="C:\Miniconda37"
+  IF "%_arch%"=="x64"
   (
-  conda install -y -c conda-forge libprotobuf=3.5.2 numpy &&
-  set "PATH=%PATH%;%CONDA_PREFIX%\Library\bin"
+  IF "%CONDA_PREFIX%"=="C:\Miniconda35-x64"
+  (
+  conda install -y -c conda-forge libprotobuf=3.5.2 numpy  
   )
   ELSE
   (
-  IF "%CONDA_PREFIX%"=="C:\Miniconda36-x64"
-  (
-  conda install -y -c conda-forge libprotobuf=3.5.2 numpy=1.17.0
+  conda install -y -c conda-forge libprotobuf=3.5.2=he51fdeb_1 numpy=1.17.0
```

Comment:
Why don't use the latest libprotobuf?

Reply:
Because latest one is built as a shared lib... Current change to appveyor simply forces it use the package version which it was using earlier and not take the latest update...

The ideal change to this CI pipeline should be to use our recommended way .i.e. build protobuf locally instead of pulling from conda-forge... but this change is a bigger one and will need all important stakeholders to agree which means it will take longer... Since right now all the PRs are blocked I want to push this change and then open discussion for the above mentioned change

Reply:
But the 3.5.x version has a serious bug on Windows: https://github.com/protocolbuffers/protobuf/pull/4777 
https://github.com/protocolbuffers/protobuf/issues/4773

Reply:
Well appveyor CI was already using 3.5.2 version... as I mentioned the aim of this PR for appveyor CI is to just force it its old behavior. Its good that you pointed this out... I am starting another thread to get a consensus on the right fix for this scenario. From my perspective the right fix is to simply build protobuf locally so that we have a better control... relying on conda-forge for protobuf has proven to be very painful so far,

Reply:
>  simply build protobuf locally
But in conda, you shouldn't do that. Because people can install protobuf from conda, and they can't safely build onnx if onnx has a local protobuf lib, in such case you don't know which protobuf lib/dll/header files will be used.